### PR TITLE
[web-console] Update README.md

### PR DESCRIPTION
**Pull request by AshishKapoor**
#### Added a step to compile scss files before running the web console. 

To avoid the following error:
```
WARNING in ./src/views/home-view/home-view.scss (./node_modules/css-loader/dist/cjs.js!./node_modules/postcss-loader/src??postcss!./node_modules/sass-loader/dist/cjs.js!./src/views/home-view/home-view.scss)
Module Warning (from ./node_modules/postcss-loader/src/index.js):
Warning

(37:3) grid-gap only works if grid-template(-areas) is being used
 @ ./src/views/home-view/home-view.scss 1:14-193 20:4-31:5 23:25-204
 @ ./src/views/home-view/home-view.tsx
 @ ./src/views/index.ts
 @ ./src/console-application.tsx
 @ ./src/entry.ts

ERROR in ./src/entry.scss (./node_modules/css-loader/dist/cjs.js!./node_modules/postcss-loader/src??postcss!./node_modules/sass-loader/dist/cjs.js!./src/entry.scss)
Module build failed (from ./node_modules/sass-loader/dist/cjs.js):
SassError: File to import not found or unreadable: ../lib/react-table.
        on line 22 of /Users/ashishkapoor/Downloads/druid-master/web-console/src/entry.scss
>> @import '../lib/react-table';

   ^

 @ ./src/entry.scss 1:14-171 20:4-31:5 23:25-182
 @ ./src/entry.ts

ERROR in ./src/views/query-view/query-input/query-input.tsx
Module not found: Error: Can't resolve '../../../../lib/sql-docs' in '/Users/ashishkapoor/Downloads/druid-master/web-console/src/views/query-view/query-input'
 @ ./src/views/query-view/query-input/query-input.tsx 10:17-52
 @ ./src/views/query-view/query-view.tsx
 @ ./src/views/index.ts
 @ ./src/console-application.tsx
 @ ./src/entry.ts

ERROR in ./src/ace-modes/dsql.js
Module not found: Error: Can't resolve '../../lib/sql-docs' in '/Users/ashishkapoor/Downloads/druid-master/web-console/src/ace-modes'
 @ ./src/ace-modes/dsql.js 25:21-50
 @ ./src/entry.ts
```

<hr>

This PR has:
- [x] been self-reviewed.
- [x] added documentation for new or modified features or behaviors.

<hr>

##### Key changed/added classes in this PR
 * Updated web-console readme file.

Sincerely
Ashish Kapoor

### CODE REVIEW

File: web-console/README.md [hunk 1]
```diff
@@ -25,7 +25,9 @@ This is the unified Druid web console that servers as a data management layer fo
 
 1. You need to be withing the `web-console` directory
 2. Install the modules with `npm install`
-3. Run `npm start` will start in development mode and will proxy druid requests to `localhost:8888`
+3. Run `npm run compile` to compile the scss files
```
Comment (suneet-s) [inline web-console/README.md:28]: I don't remember having to do this step when I ran the web console locally. Can you describe what wasn't working with the instructions before this change

Comment (AshishKapoor) [reply web-console/README.md:28]:
```
Build failed with error code: 1
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.9 (node_modules/fsevents):
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.9 install: `node install`
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: Exit status 1

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! node-sass@4.13.1 postinstall: `node scripts/build.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the node-sass@4.13.1 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/ashishkapoor/.npm/_logs/2020-09-14T15_36_01_509Z-debug.log
➜  web-console 
```

Comment (AshishKapoor) [reply web-console/README.md:28]: Wait, @suneet-s  Let me verify this again. I believe it's because there's no Xcode on the machine in use.

Comment (AshishKapoor) [reply web-console/README.md:28]: There you go!

```
WARNING in ./src/views/home-view/home-view.scss (./node_modules/css-loader/dist/cjs.js!./node_modules/postcss-loader/src??postcss!./node_modules/sass-loader/dist/cjs.js!./src/views/home-view/home-view.scss)
Module Warning (from ./node_modules/postcss-loader/src/index.js):
Warning

(37:3) grid-gap only works if grid-template(-areas) is being used
 @ ./src/views/home-view/home-view.scss 1:14-193 20:4-31:5 23:25-204
 @ ./src/views/home-view/home-view.tsx
 @ ./src/views/index.ts
 @ ./src/console-application.tsx
 @ ./src/entry.ts

ERROR in ./src/entry.scss (./node_modules/css-loader/dist/cjs.js!./node_modules/postcss-loader/src??postcss!./node_modules/sass-loader/dist/cjs.js!./src/entry.scss)
Module build failed (from ./node_modules/sass-loader/dist/cjs.js):
SassError: File to import not found or unreadable: ../lib/react-table.
        on line 22 of /Users/ashishkapoor/Downloads/druid-master/web-console/src/entry.scss
>> @import '../lib/react-table';

   ^

 @ ./src/entry.scss 1:14-171 20:4-31:5 23:25-182
 @ ./src/entry.ts

ERROR in ./src/views/query-view/query-input/query-input.tsx
Module not found: Error: Can't resolve '../../../../lib/sql-docs' in '/Users/ashishkapoor/Downloads/druid-master/web-console/src/views/query-view/query-input'
 @ ./src/views/query-view/query-input/query-input.tsx 10:17-52
 @ ./src/views/query-view/query-view.tsx
 @ ./src/views/index.ts
 @ ./src/console-application.tsx
 @ ./src/entry.ts

ERROR in ./src/ace-modes/dsql.js
Module not found: Error: Can't resolve '../../lib/sql-docs' in '/Users/ashishkapoor/Downloads/druid-master/web-console/src/ace-modes'
 @ ./src/ace-modes/dsql.js 25:21-50
 @ ./src/entry.ts
```

Comment (suneet-s) [reply web-console/README.md:28]: Thanks @AshishKapoor I haven't run into this issue when I ran through these instructions. The update seems reasonable to me.

I'll let someone more familiar with front-end dev chime in, I've only compiled and run the web console locally, a handful of times.

Comment (AshishKapoor) [reply web-console/README.md:28]: That makes sense, thanks! @suneet-s.

Comment (czyzykowski) [reply web-console/README.md:28]: I can confirm that without running `npm run compile` `npm start` fails with the mentioned errors. And I do have Xcode.

Comment (AshishKapoor) [reply web-console/README.md:28]: Please take a look at this one too. @vogievetsky 🙏🏼

**Review by nishantmonu51 — APPROVED** [review]
👍 , LGTM

**Pull request merged**